Library/Services/WorkflowManager.php

```diff
@@ -17,6 +17,7 @@ class WorkflowManager extends Component implements WorkflowManagerInterface
     const STATUS_RUNNING    = 'RUNNING';
 
     private $config = null;
+    private $workflowProcessId = null;
```

Comment:
not sure it's ok, I mean, at one time, the manager handle several workflow execution...